### ISSUE TITLE
Form RTL mode issues

### ISSUE THREAD

**Issue by dioslaska** [acidb/mobiscroll#392]
In right-to-left languages there are a few rendering issues:

1.  Align Switch, Checkbox, Radio and Stepper to the other side
2. Switch is not rendered correctly
3. Stepper is not rendered correctly with the value on the left or right

**Comment by stehlo** [comment]
4. Segmented (in case of 2 elements) renders with rounded corners in the middle, not on the outside.

**Comment by stehlo** [comment]
-- **5. Segmented elements are using labels incorrectly.**

The following code creates the labels **incorrectly**:
(The labels are duplicated above the buttons.)
```html
      <div class="mbsc-input">
        <span data-l10n="Temperature units">وحدات درجة الحرارة</span>
        <div>
          <label for="units_c" dir="ltr">°C</label>
          <input id="units_c" type="radio" data-role="segmented" name="units" value="c" checked/>
        </div>
        <label>
          <label for="units_f" dir="ltr">°F</label>
          <input id="units_f" type="radio" data-role="segmented" name="units" value="f"/>
        </label>
      </div>
```

The following code creates the labels **correctly** in the RTL environment because it is still rather simple:
```html
      <div class="mbsc-input">
        <span data-l10n="Temperature units">وحدات درجة الحرارة</span>
        <div dir="ltr">
          °C
          <input id="units_c" type="radio" data-role="segmented" name="units2" value="c" checked/>
        </div>
        <label dir="ltr">
          °F
          <input id="units_f" type="radio" data-role="segmented" name="units2" value="f"/>
        </label>
      </div>
```

However, the following code creates the labels again **incorrectly** and it can't be fixed in a way like the first way above:
(The labels are duplicated above the buttons.)
```html
      <div class="mbsc-input">
        <span data-l10n="Temperature units">وحدات درجة الحرارة</span>
        <div>
          <label for="units_c"><span dir="ltr">°C</span> درجة الحرارة</label>
          <input id="units_c" type="radio" data-role="segmented" name="units3" value="c" checked/>
        </div>
        <label>
          <label for="units_f"><span dir="ltr">°F</span> درجة الحرارة</label>
          <input id="units_f" type="radio" data-role="segmented" name="units3" value="f"/>
        </label>
      </div>
```

**Comment by stehlo** [comment]
Sounds good. Thank you.

**Comment by dioslaska** [comment]
Fixed in 3.0.0-beta5
